src/react-queries/queries/common.ts

```diff
@@ -90,43 +89,15 @@ export const common = createQueryKeys('common', {
   }),
   evmFees: ({
     data,
-    wallet,
     accountNumber,
     to,
-    from,
     value,
-    feeAsset,
-    feeAssetMarketData,
-  }: {
-    to: string
-    // Only used to make the queryKey unique by from address, since wallet doesn't serialize and is excluded from the queryKey
```

Comment:
Actually not needed, we now have `chainId` (which is needed for the hook to get the right adapter), and the combination of `chainId` + `accountNumber` is effectively all we need to avoid collisions